i18n/de/docusaurus-plugin-content-docs/current/workspace/projects.md

```diff
@@ -2,18 +2,18 @@
 sidebar_position: 2
 ---
 
-# Projects
+# Projekte
 
-The **Projects** page allows users to **create, manage, and share** projects. The page provides users with an organized view of their projects, categorized into personal projects, team projects, and projects shared with the whole organization. In addition, projects can be organized into **folders** and can be **filtered** and **sorted** based on alphabetical order, date of creation, or date of last update. It is also possible to **delete projects** and **edit their metadata**. 
+Die **Projekte**-Seite ermöglicht es Benutzern, Projekte zu **erstellen, zu verwalten und zu teilen**. Die Seite bietet den Nutzern eine übersichtliche Darstellung ihrer Projekte, kategorisiert in persönliche Projekte, Teamprojekte und Projekte, die mit der gesamten Organisation geteilt werden. Darüber hinaus können Projekte in **Ordnern** organisiert und **gefiltert** und **sortiert** werden, basierend auf alphabetischer Reihenfolge, Erstellungsdatum oder Datum der letzten Aktualisierung. Es ist auch möglich, **Projekte zu löschen** und deren **Metadaten zu bearbeiten**.
 
 <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
   <img src={require('/img/workspace/projects/projects_general.png').default} alt="Projects Page in Workspace of GOAT" style={{ maxHeight: "auto", maxWidth: "auto", objectFit: "cover"}}/>
 
 </div> 
 
 
-## Creating a new project
-Follow these steps to add a new project to your GOAT workspace:
+## Neues Projekt erstellen
+Folgen Sie diesen Schritten, um ein neues Projekt zu Ihrem GOAT-Arbeitsbereich hinzuzufügen:
 
 <div class="step">
```

Comment:
The text within the all the "step" sections here and below can be translated too